sagemaker-inference-recommender/xgboost/xgboost-inference-recommender.ipynb

```diff
@@ -0,0 +1,793 @@
+{
```

Comment:
<p>Since you give detailed instructions for notebook instances, can you mention whether this notebook is supported in Studio as well?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3073/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Added a kernel recommendation for SM Studio</p>